.github/workflows/build-hotfix.yml

```diff
@@ -65,3 +66,20 @@ jobs:
           git commit -m "chore(dependencies): Autobump oss clouddriver version"
           git pull
           git push
+      # Adding OSS docker build for SNAP
+      - name: Publish Release
+        if: startsWith(github.ref, 'refs/heads/release-')
+        run: |
+          ./gradlew final test docker dockerPush $GRADLE_ARGS
```

Comment:
This command is for armory repo extensions, this repo doesn't have any gradle docker integration. I suggest to manually run all commands needed to publish the image locally to make sure they work and then adjust the workflow file.

Reply:
Changed to using the docker commands directly,